client/src/components/board.js

```diff
@@ -52,11 +52,29 @@ class Board extends React.Component {
   }
 
   render() {
+    const copyToClipboard = function copyToClipboard() {
+      document.getElementById('urlInput').value = window.location.href;
```

Comment:
Yea, I think having the link display in a text box that they can copy is a lot simpler for the MVP, and it's a familiar concept to have to copy/paste things by highlighting.